superdesk/commands/delete_text_doc.py

```diff
@@ -0,0 +1,27 @@
+import superdesk
+
+
+class DeleteDocCommand(superdesk.Command):
+    '''
+
+    Delete a Text Archive document from both Mongodb and ElasticSearch by supplying a GUID
+
+    manage.py app:deletedoc
+        -i urn:newsml:localhost:2016-09-05T11:54:49.869091:6bdbd02b-f6c5-46d3-bc53-4c1fc74252af
+
+    '''
+    option_list = [
+        superdesk.Option('--guids', '-i', dest='guids', required=True,
+                         help='pass guids for articles to delete')
+    ]
+
+    def run(self, guids):
+        guids = guids.split(",")
```

Comment:
We might want to delete 100s of documents due to contractual agreement.  A `file` options could  be handy for batch delete. We can do `mongoexport` of guids and feed it to the `deletecommand`. 

Moreover, it would be nice to have an option for `batch size` for deleting documents.